main.scss

```diff
@@ -7,17 +7,30 @@
 @import 'src/scss/functions';
 @import 'src/scss/share';
 
-@if (not $o-share-is-silent) {
-	@include oShareBase;
+@mixin oShare($opts: (
+  'icons': $_o-share-icons,
+  'vertical': true,
+  'inverse': true
+)) {
+  $icons: map-get($opts, 'icons');
+  $vertical-enabled: map-get($opts, 'vertical');
+  $inverse-enabled: map-get($opts, 'inverse');
+
+  @include _oShareBase;
 
-	// run through all the different icons available
-	@each $icon in $_o-share-icons {
-		@include oShareActionIcon($icon);
-	}
+  @each $icon in $_o-share-icons {
+	  @include _oShareActionIcon($icon);
+  }
 
-	@include oShareInverse;
+  @include _oShareInverse;
```

Comment:
Don't forget the `$inverse-enabled` conditional!

Reply:
woops!